components/api_request_helper/api_request_helper.cc

```diff
@@ -202,25 +227,26 @@ void APIRequestHelper::OnResponse(
 
   url_loaders_.erase(iter);
   if (!response_body) {
-    std::move(callback).Run(APIRequestResult(
-        response_code, "", std::move(headers), error_code, final_url));
+    std::move(callback).Run(APIRequestResult(response_code, "", base::Value(),
+                                             std::move(headers), error_code,
+                                             final_url));
     return;
   }
   auto& raw_body = *response_body;
   if (conversion_callback) {
     auto converted_body = std::move(conversion_callback).Run(raw_body);
     if (!converted_body) {
       std::move(callback).Run(APIRequestResult(
-          422, std::move(raw_body), std::move(headers), error_code, final_url));
+          422, "", base::Value(), std::move(headers), error_code, final_url));
```

Comment:
Non-sanitized raw_body could potentially leak to caller. Now just return empty body.